.gitignore

```diff
@@ -13,10 +13,4 @@ wiki-wishlist
 *.sublime-workspace
 .editorconfig
 .idea
-lib
-/plugins
-.vscode/*
-!.vscode/settings.json
```

Comment:
I would keep all `.vscode` entries in `.gitignore`.

Reply:
...or only `.vscode`, of course

Reply:
Guess that just happened. Feel free to change it back!  I won't have time to edit anything about the PR for the next few days.